Lang/zh-cn.cfg

```diff
@@ -0,0 +1,986 @@
+// Auto generated by KSPDev Localization tool at: 9/2/2017 1:36:15 AM
+// Total strings: 231
+// Total words: 1188
+Localization
+{
+	en-us
```

Comment:
This section should be named "zh-cn".

Reply:
Oh, yes, I forgot the changes after copying

Reply:
You can modify it after merging.

Reply:
In general it's better to keep the pull requests atomic. But no worries, I'l merge and add a fix.